EU4toV2/Source/Configuration.h

```diff
@@ -56,7 +56,8 @@ class Configuration: commonItems::parser
 		double getMaxLiteracy() { return MaxLiteracy; }
 		std::string getRemovetype() { return Removetype; }
 		double getLibertyThreshold() { return libertyThreshold; }
-		bool getConvertPopTotals() { return convertPopTotals; }
+		std::string getPopShaping() { return popShaping; }
```

Comment:
A nit: this would probably better be a class enum, but there's so many bad examples in this codebase...